Template/Tag/BingUETTag.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\TagManager\Template\Tag;
+
+use Piwik\Settings\FieldConfig;
+use Piwik\Validators\NotEmpty;
+use Piwik\Validators\NumberRange;
+
+class BingUETTag extends BaseTag
+{
+    public function getIcon()
+    {
+        return 'plugins/TagManager/images/icons/bing.svg';
+    }
+
+    public function getParameters()
+    {
+        return array(
+            $this->makeSetting('bingAdID', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'ID';
+                $field->uiControl = FieldConfig::UI_CONTROL_TEXT;
+                $field->description = '';
```

Comment:
Is there an easy way to get the ID? PR looks good otherwise (haven't tested though)

Reply:
Ah, I forgot the description, because I couldn't think of a great wording.

The ID is quite visible in the table where new tags can be created and managed and is also displayed before the tracking code in the modal.